.github/workflows/nightly.yml

```diff
@@ -239,6 +239,10 @@ jobs:
         run: |
           # It's a temporary fix, to be removed when the issue will be resolved
           pacman -U --noconfirm https://repo.msys2.org/mingw/ucrt64/mingw-w64-ucrt-x86_64-exiv2-0.27.7-1-any.pkg.tar.zst
+      - name: Install the latest still safe to display cyrillic texts version of pango
```

Comment:
I've probably missed it, what the issue? Maybe open one if not already and reference it here?